docs/differenceBetweenPBSGo-and-Java.md

```diff
@@ -1,36 +1,34 @@
 # Differences Between Prebid Server Go and Java
 
-October 26, 2018
+January 24, 2019
 
 The sister Prebid Server projects are both busy and moving forward at different paces on different features. Sometimes a feature may exist in one implementation
 and not the other for an interim period. This page tracks known differences that may persist for longer than a couple of weeks.
 
 ## Feature Differences
 
-1) PBS-Java supports Currency conversion. PBS-Go [issue 280](https://github.com/prebid/prebid-server/issues/280). PBS-Java [PR 22](https://github.com/rubicon-project/prebid-server-java/pull/22)
+1) PBS-Java supports Currency conversion. PBS-Go currently has it disabled as it's still under dev [issue 280](https://github.com/prebid/prebid-server/issues/280), [issue 760](https://github.com/prebid/prebid-server/pull/760). PBS-Java [PR 22](https://github.com/rubicon-project/prebid-server-java/pull/22)
```

Comment:
Pls notice PBS-Go has Currency conversion implemented but disabled by default.

Reply:
I've also added a note from their doc that it's due to this feature still being under dev.

Reply:
ok, cool.